src/main/java/io/strimzi/kafka/quotas/StorageChecker.java

```diff
@@ -4,72 +4,47 @@
  */
 package io.strimzi.kafka.quotas;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.io.IOException;
 import java.io.UncheckedIOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.List;
-import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.Consumer;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * Periodically reports the total storage used by one or more filesystems.
  */
 public class StorageChecker implements Runnable {
     private static final Logger log = LoggerFactory.getLogger(StorageChecker.class);
 
-    private final Thread storageCheckerThread = new Thread(this, "storage-quota-checker");
-    private final AtomicBoolean running = new AtomicBoolean(false);
     private final AtomicLong storageUsed = new AtomicLong(0);
 
-    private volatile long storageCheckIntervalMillis;
     private volatile List<Path> logDirs;
     private volatile Consumer<Long> consumer;
 
-    void configure(long storageCheckIntervalMillis, List<Path> logDirs, Consumer<Long> consumer) {
-        this.storageCheckIntervalMillis = storageCheckIntervalMillis;
+    void configure(List<Path> logDirs, Consumer<Long> consumer) {
         this.logDirs = logDirs;
         this.consumer = consumer;
     }
 
-    void startIfNecessary() {
```

Comment:
I don't think there is a strong need to maintain the API here but can easily restore the method signatures (and mark deprecated) if desired.

Reply:
You mean as in _backwards compatibility_? I think this is an implementation detail. The API would be the configuration of the plugin. So I think it is fine to remove this.

Reply:
Yes, backwards compatibility was my concern.